vhost/src/vhost_user/message.rs

```diff
@@ -987,6 +991,45 @@ impl VhostUserMsgValidator for VhostUserTransferDeviceState {
     }
 }
 
+// Bit mask for flags in VhostUserMMap struct
+bitflags! {
+    #[derive(Copy, Clone, Debug, Eq, PartialEq, Default)]
+    /// Flags specifying access permissions of memory mapping of a file
+    /// Use VhostUserMMapFlags::default() for a read-only mapping
+    pub struct VhostUserMMapFlags: u64 {
+        /// Read-write permission
```

Comment:
nit: `/// Read-write permission (0: read-only, 1: read-write)`

Reply:
ah, I thought was a bitfields.

Or maybe `/// Read-write permission (if not set, it's read-only)`, but I see we already have something in the struct doc, so really not a strong opinion on this, feel free to do nothing ;-)